packages/react-core/src/components/Form/FormGroup.tsx

```diff
@@ -107,7 +110,13 @@ export const FormGroup: React.FunctionComponent<FormGroupProps> = ({
         </div>
       )}
 
-      <div className={css(styles.formGroupControl, isInline && styles.modifiers.inline)}>
+      <div
+        className={css(
+          styles.formGroupControl,
+          isInline && styles.modifiers.inline,
+          isStacked && styles.modifiers.stack
```

Comment:
wdyt about `isStack`? It matches the modifier name and the layout and usage of `pf-m-stack` in the sidebar. We do have `pf-m-stacked` in the tile component but I'm wondering if we should change that. @mcarrano do you know if we have any kind of guideline for this?

Reply:
Not sure I understand the question. Do you mean when to use "stack" vs "stacked?"

Reply:
@mcarrano yes, I noticed that we may have an inconsistent use of stack & stacked.

* `pf-l-stack` layout 
* `pf-m-stack` variation of the sidebar layout
* `pf-m-stack` variation of form control groups (to stack inputs on top of one another)
* `pf-m-stacked` variation for tiles

Reply:
I see. Yes, it would be nice to be consistent. @mcoker what are the implications of changing this in Tile? Won't that potentially break people?

Reply:
@mcarrano it's a breaking change, yes. We'll just make sure the usage in this PR is consistent with our intent, then update any other component we need in a breaking change release.